internal/worker/server.go

```diff
@@ -613,6 +613,7 @@ func (s *Server) requestJob(ctx context.Context, arch string, jobTypes []string,
 		jobId, token, depIDs, jobType, args, err = s.jobs.Dequeue(dequeueCtx, jts, channels)
 	}
 	if err != nil {
+		logrus.Errorf("dequeuing job failed: %v", err)
```

Comment:
I think we want to check for `err` not being a timeout? Otherwise, there will be a ton of entries.

Reply:
good point, fixed